test/Microsoft.AspNetCore.Mvc.Test/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+{
+  "shadowCopy": false
```

Comment:
Why is this file needed?

Reply:
deps file on desktop. I thought the shadow copying stuff was fixed, apparently not though